packages/playground-server/.env-cmdrc.local

```diff
@@ -0,0 +1,14 @@
+{
+  "development": {
+    "DB_ENGINE": "pg",
+    "DB_CONNECTION_STRING": "postgresql://playground_user:GFnggtxysjRjKJ3AN8g6cTYg@104.248.239.2:5432/playground_db",
```

Comment:
... staging env password shouldn't just be in plain text here

Reply:
Deleted it